index.php

```diff
@@ -100,6 +100,15 @@ function getLastRevision($distro) {
 # or
 sudo ./llvm.sh all
 </pre>
+
+Additionally, there are options to specify the mirror and distro version to use, for example:
+<pre>
+sudo ./llvm.sh -m https://mirrors.tuna.tsinghua.edu.cn/llvm-apt -d Ubuntu_22.04
```

Comment:
As https://github.com/opencollab/llvm-jenkins.debian.net/pull/19#discussion_r870324886 pointed out, maybe removing `sudo` here is more appropriate?

Reply:
But it is not directly running downloaded script, and there are other occurrences in the page that has sudo 🤔